pkg/virtual/workspaces/registry/rest.go

```diff
@@ -203,12 +192,22 @@ func withoutGroupsWhenPersonal(user user.Info, scope string) user.Info {
 	return user
 }
 
+func (s *REST) extractOrg(ctx context.Context) (orgName string, org Org, err error) {
```

Comment:
orgClusterName

Reply:
and *Org everywhere. This struct is big already. No need for call-by-value.

Reply:
Fixed in commit 2ebfda87ecb2ea9d9c303301deb6296999938a7b